Explorer/Assets/DCL/Utilities/Extensions/FluentExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace DCL.Utilities.Extensions
+{
+    public static class FluentExtensions
```

Comment:
What is this for?

Reply:
ah, I was trying different approaches for refactoring and this was a part. But then I removed its usage, since it didn't work in my case. So, I will remove it.